src/Uno.UI/UI/Xaml/UIElement.netstd.cs

```diff
@@ -196,5 +196,55 @@ private void OnChildRemoved(UIElement child)
 
 		internal Point GetPosition(Point position, UIElement relativeTo)
 			=> TransformToVisual(relativeTo).TransformPoint(position);
+
+#if DEBUG
+
+		/// <summary>
+		/// Convenience method to find all views with the given name.
+		/// </summary>
+		public FrameworkElement[] FindViewsByName(string name) => FindViewsByName(name, searchDescendantsOnly: false);
```

Comment:
`IEnumerable<FrameworkElement>` ?

Reply:
the only time you can use this is in debugging, where it's more convenient to have an array and you don't care about perf ;)

Reply:
Agree. Debugging with `IEnumerable`s is a challenge. Thanks :)